### ISSUE TITLE
Add "unkown" to cast role if meta is missing

### ISSUE THREAD

**Issue by candry7731** [jellyfin-archive/jellyfin-roku-legacy#1032]
**Is your feature request related to a problem? Please describe.**
Currently, if a cast member is missing role metadata then the secondary text is "as" then a blank

**Describe the solution you'd like**
Like to add the work "unknown" after the "as" if there is no role metadata

**Additional context**

Missing role data
![image](https://user-images.githubusercontent.com/83685828/219751311-a50ea39c-c53b-413e-8d41-80745aee3d35.png)

Add "unknown" (just the text, not the background)
![image](https://user-images.githubusercontent.com/83685828/219751543-bf1e784e-c53e-4308-8158-b96729f5efad.png)

**Comment by cewert** [comment]
Maybe we should also move the "unknown"s to the end of the Cast & Crew list? Your screenshot shows a list of 19 people and the first 5 are missing a picture and don't have a known role. If possible, I think we should sort the list based on available metadata like picture and role.

**Comment by candry7731** [comment]
@cewert I like that Idea. I wonder how we could sort it?

**Comment by cewert** [comment]
I checked a few movies of mine and all the ones I checked were sorted just like the web client - with actors at the front of the list sorted by billing i think followed by directors etc.. So I'm thinking this is a fluke and your metadata is just messed up for that movie. What does the cast for that movie look like when you view it on the web client?

If I remember correctly brightscright arrays don't support sorting so we'd have to hack something together. It's totally possible but would be ugly and slow I think. Hopefully its just a metadata issue.
